docker/docker-compose.mysql-ssl-server-ssl-connector_transparent.yml

```diff
@@ -123,10 +123,8 @@ services:
             --tls_cert=/ssl.server/acra-server.crt
             --tls_key=/ssl.server/acra-server.key
             --tls_db_sni=mysql
-            --acraconnector_transport_encryption_disable
             --encryptor_config_file=/acra-encryptor.yaml
-            --tls_auth=1
-            -v
+            -d
```

Comment:
Why do we need to start it as a daemon instead of no-detach mode, which is expectable docker-way?

Reply:
This parameter stands for debug mode in acra-server and just an development artifact, thx for noticing, will fix it